edb/server/protocol/auth_ext/config.py

```diff
@@ -21,14 +21,13 @@
 
 
 class UIConfig:
-    def __init__(
-        self,
-        app_name: Optional[str] = None,
-        logo_url: Optional[str] = None,
-        dark_logo_url: Optional[str] = None,
-        brand_color: Optional[str] = None,
-    ) -> None:
-        self.app_name = app_name
-        self.logo_url = logo_url
-        self.dark_logo_url = dark_logo_url
-        self.brand_color = brand_color
+    redirect_to: str
+    redirect_to_on_signup: Optional[str]
+    app_name: Optional[str]
+    logo_url: Optional[str]
+    dark_logo_url: Optional[str]
+    brand_color: Optional[str]
+
+
+class ProviderConfig:
+    name: str
```

Comment:
I'm not sure I _quite_ understand the difference here, but it seemed weird to have an initializer on this class when we're really just using it so we can cast it as an interface/type.

Reply:
:+1: Right, we don't need the initializer here I believe - other than the ability to instantiate this type, I don't think the initializer brings anything else differently compared to the new way you used here for typing purposes.